From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Sat, 22 Apr 2023 01:21:20 +0000
Subject: [PATCH] rusticl: Fix bindgen invocation

The deprecated `--size_t-is-usize` was removed in bindgen 0.65.1, being
enabled by default.
---
 src/gallium/frontends/rusticl/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/gallium/frontends/rusticl/meson.build b/src/gallium/frontends/rusticl/meson.build
index e53b50a9b8b8..9fa024cb8db4 100644
--- a/src/gallium/frontends/rusticl/meson.build
+++ b/src/gallium/frontends/rusticl/meson.build
@@ -92,7 +92,6 @@ rusticl_gen_args = [
 rusticl_bindgen_args = [
   '--no-convert-floats',
   '--use-array-pointers-in-arguments',
-  '--size_t-is-usize',
   '--default-enum-style', 'rust',
   '--with-derive-partialeq',
   '--with-derive-eq',
